### ISSUE TITLE
Salesforce Lightening

### ISSUE THREAD

**Issue by Mahwish7** [choppen5/TwilioSalesforceClick2Dial#7]
Hi

Is this salesforce lightening enabled? I have to use in lightening could you please advice?

Thanks

**Comment by paulbfischer** [comment]
No, I don't believe so. Salesforce Lightning uses a different set of
OpenCTI methods, which I don't think have been implemented in this code yet.

**Comment by aamir-munir** [comment]
ok 2 questions:
1. anyone working to make it available with lightning as well?
2. once I implement this can I have the same functionality in mobile app?

Thanks
